winui/Shimmer/Custom-Layout.md

```diff
@@ -0,0 +1,325 @@
+---
+layout: post
+title: Custom Layout in WinUI Shimmer control | Syncfusion
+description: Learn here all about the Custom Layout support in Syncfusion WinUI Shimmer (SfShimmer) control and more.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# Custom Layout of WinUI Shimmer (SfShimmer)
+
+The `Custom Layout` property of the SfShimmer can be used to create your shimmer view. The below code demonstrates how to customize the shimmer view using the WinUI framework elements by `Custom Layout`.
+
```

Comment:
@GilbertmanojSF3971 
# Custom layout of WinUI Shimmer (SfShimmer)

The `CustomLayout` property empowers you to design your own shimmer type and giving you full control over create your desired shimmer type.

## How to create a custom shimmer type

You can create a custom shimmer layout by setting the `CustomLayout` property of the SfShimmer control. Below is a code example that demonstrates how to achieve this:

Reply:
@Manivannan-E Sir , Changed the font size of the title as well as the content in the PR.